### ISSUE TITLE
Problematic setting avatar function

### ISSUE THREAD

**Issue by youselfcan** [Merubokkusu/Discord-S.C.U.M#34]
I tried with 10+ tokens, after I put an avatar with discum.py, I installed it with downloading the zip and then installing it with pip, even If I manually join a guild, instantly I get blocked and discord automatically asks for verification. Also after I set an avatar, I cant seem to send messages to a channel, says that I'm unauthorized. I've tried to get my profile via info() and getProfile(), didnt work. I tried to put some delay, didnt work neither.
note:I'm using freshly generated tokens, they dont have mails and passwords. Maybe it's because I'm just using tokens, but you can change your avatar manually without setting email or password.

It would be nice to change avatars automatically. If you need more info,

**Comment by youselfcan** [comment]
I mean, I can use setAvatar and set my avatar, but the problem is even If I manually join a guild on actual discord app, it blocks me out for some reason. Joining guilds with discum has no problem for me, just these steps cause problems:
1- set avatar with discum // I can set an avatar with discum without a problem
2- join a guild // doesnt matter If I join a guild via discord app or discum.py, both blocks me out.

**Comment by youselfcan** [comment]
Nope, If I set my avatar manually I'm not getting locked out, I tried setting an avatar manually then joining a guild manually, its working. I also tried setting an avatar manually and joining with discum, it works too.

**Comment by NaC-L** [comment]
Yup, I tried getting my profile then setting an avatar, same problem. Its interesting, after I set my avatar, I can send messages (sometimes I can send with no problem), react to stuff, but cant join a guild. I've also tried using getInfoFromInviteCode before joinGuild, no luck. 
__________
arandomnewaccount:
moving discussion from this issue to https://github.com/Merubokkusu/Discord-S.C.U.M/issues/66
